astropy/coordinates/builtin_frames/skyoffset.py

```diff
@@ -177,3 +177,10 @@ def represent_as(self, base, s='base', in_frame_units=False):
         data = super().represent_as(base, s, in_frame_units=in_frame_units)
         self._set_skyoffset_data_lon_wrap_angle(data)
         return data
+
+    def __reduce__(self):
```

Comment:
I think I would pass on origin as a positional argument to `_skyoffset_reduce`, though it is not a big deal at all.
```
    def __reduce__(self):
        return (_skyoffset_reducer, (self.origin,), self.__dict__)
```